src/components/charts/chart-composed/chart-composed.md

```diff
@@ -108,6 +108,7 @@ const lineWithDots = config.columns && config.columns.lineWithDots.map(column =>
     height={500}
     loading={state.loading}
     onLegendChange={handleLegendChange}
+    showUnit={true}
```

Comment:
the ={true} part is not needed if is a true boolean. You can write just showUnit

Reply:
Thanks, I forgot about that, fixed :)